src/asl.ts

```diff
@@ -854,7 +870,7 @@ export class ASL {
 
         const initializer: ASLGraph.SubState | ASLGraph.NodeState = (() => {
           /**ForInStmt
-           * Assign the value to $.0__[variableName].
+           * Assign the value to $.fnls__0__[variableName].
```

Comment:
Why change?

Reply:
breaks :(

That issue I tweeted about, Intrinsic functions cannot handle json paths that start with a number, underscore, etc, so better to just use valid paths.